aten/src/ATen/xpu/profiler/XPUProfilerMacros.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <libkineto.h>
+#include <output_base.h>
+#include <time_since_epoch.h>
+
+#include <pti/pti_view.h>
+
+#include <c10/xpu/XPUMacros.h>
+
+namespace at::kineto_plugin::xpu {
+
+static std::string errorToMsg(const pti_result& result) {
+  std::string preMsg("PTI-based kineto profiler for XPU got error(");
+  std::string Msg;
+  switch (result) {
+    case PTI_STATUS_END_OF_BUFFER:
+      Msg = "end of buffer reached";
+    case PTI_ERROR_NOT_IMPLEMENTED:
+      Msg = "functionality not implemented";
+    case PTI_ERROR_BAD_ARGUMENT:
+      Msg = "error code for invalid arguments";
+    case PTI_ERROR_NO_CALLBACKS_SET:
+      Msg = "error due to no callbacks set via ptiViewSetCallbacks";
+    case PTI_ERROR_EXTERNAL_ID_QUEUE_EMPTY:
+      Msg =
+          "empty external ID-queue while working with PTI_VIEW_EXTERNAL_CORRELATION";
+    case PTI_ERROR_DRIVER:
+      Msg = "driver error";
+    case PTI_ERROR_TRACING_NOT_INITIALIZED:
+      Msg =
+          "installed driver requires tracing enabling with setting environment variable ZE_ENABLE_TRACING_LAYER to 1";
+    case PTI_ERROR_L0_LOCAL_PROFILING_NOT_SUPPORTED:
+      Msg = " no Local profiling support in the installed driver";
+    case PTI_ERROR_INTERNAL:
+      Msg = "internal error";
+    default:
+      Msg = "unknown error";
+  }
+  std::string sufMsg = "). Please file issue to IPEX";
+  return preMsg + Msg + sufMsg;
+}
+
+#define ptiCheck(returnCode)                          \
```

Comment:
rename to `AT_XPU_PTI_CHECK`? following pytorch code style, these letters should be capitalized better.

Reply:
ok

Reply:
Let's remove the AT_ prefix from this macro.

Reply:
ok